indy_node/test/upgrade/test_node_control_tool_resolves_dep_top_level.py

```diff
@@ -53,8 +55,10 @@ def mock_get_info_from_package_manager(package):
     monkeypatch.setattr(NodeControlUtil, 'update_package_cache', lambda *x: None)
     monkeypatch.setattr(NodeControlUtil, '_get_info_from_package_manager',
                         lambda x: mock_get_info_from_package_manager(x))
+    monkeypatch.setattr(NodeControlUtil, 'get_sys_holds',
```

Comment:
Can we extend the test to depend on python3-indy-crypto which depends on libindy-crypto?

Reply:
done